### ISSUE TITLE
Accessibility - 2.1.1 Keyboard

### ISSUE THREAD

**Issue by phucduong86** [CodingTrain/website#1228]
When viewing the website in mobile mode, mobile navigation toggle is not accessible by keyboard and user is not able to access the navigation links. Recommend implement keyboard access method (either by using button/anchor element or javascripts)

**Comment by phucduong86** [comment]
Resolved with #1380
